EXAMPLE_README.md

```diff
@@ -33,101 +33,93 @@
 
 ## Project Overview
 
-1️⃣ [Trello Board](🚫add link to trello board here)
+1️⃣ [Trello Board](https://trello.com/b/f5IBtPhV/labs25hrfbrannan)
 
-1️⃣ [Product Canvas](🚫add link to product canvas here)
+1️⃣ [Product Canvas](https://whimsical.com/4Gg7cNjtFfmcAXKDpRu8oJ)
 
-1️⃣ [UX Design files](🚫add link to design files here) - 🚫 delete if not applicable
+1️⃣ [UX Design files](https://www.figma.com/file/0WO6A6jQw3o00KSqcUv1Vm/PoliceForceApp?node-id=0%3A1) - 🚫 delete if not applicable
 
-🚫 Replace lorem ipsum with a description of your project
-
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
-
-Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+Single page application that maps police brutality based on scraping data from Twitter and Reddit.  It includes a timeline view, visual
+map and feed page.  Intended for journalists, activists and lawyers.
```

Comment:
Let's use the phrase 'use of force' instead of brutality. Social media instead of reddit and twitter.